### PR TITLE
Box - created docs

### PR DESCRIPTION
**Pull request by abaumgarten**
## Status
- [] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/29973

## Description
Box integration was missing docs. Generated docs for this integration.
 
## Minimum version of Demisto
- [x] 5.0.0
- [ ] 5.5.0
- [ ] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

### CODE REVIEW

File: Packs/Box/Integrations/README.md [hunk 1]
```diff
@@ -0,0 +1,519 @@
+Manage Box users
+This integration was integrated and tested with version xx of Box
```
Comment (teizenman) [inline Packs/Box/Integrations/README.md:2]: Same as with GoogleDocs

**Review by teizenman — CHANGES_REQUESTED** [review]

File: Packs/Box/Integrations/README.md (hunk 1 above)
Comment (abaumgarten) [reply Packs/Box/Integrations/README.md:2]: You right :)

**Review by teizenman — APPROVED** [review]

File: Packs/Box/Integrations/README.md [hunk 2]
```diff
@@ -0,0 +1,518 @@
+Manage Box users
+## Configure Box on Cortex XSOAR
```
Comment (Itay4) [inline Packs/Box/Integrations/README.md:2]: missing here the configuration steps
you can see them in the detailed description

Comment (abaumgarten) [reply Packs/Box/Integrations/README.md:2]: Done.

File: Packs/Box/Integrations/README.md [hunk 3]
```diff
@@ -0,0 +1,535 @@
...
+2. Search for Box.
+3. Click **Add instance** to create and configure a new integration instance.
+
+| **Parameter** | **Description** | **Required** |
+| --- | --- | --- |
+| insecure | Trust any certificate \(not secure\) | False |
+
+4. Click **Test** to validate the URLs, token, and connection.
+
+## How to initialise:
```
Comment (Itay4) [inline Packs/Box/Integrations/README.md:13]: @kirbles19 @ShirleyDenkberg please review this section

Comment (abaumgarten) [reply Packs/Box/Integrations/README.md:13]: Section's header took from https://github.com/demisto/etc/issues/14446

**Review by guykeller — APPROVED** [review]

**Comment by ShirleyDenkberg** [comment]
@Itay4 I reviewed and edited the How to Initialize section in the README.

**Pull request merged**